addons/mail/models/mail_message_translation.py

```diff
@@ -0,0 +1,26 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class MessageTranslation(models.Model):
+    _name = "mail.message.translation"
+    _description = "Message Translation"
+
+    message_id = fields.Many2one(
+        "mail.message",
+        "Message",
+        required=True,
+        index=True,
+        ondelete="cascade",
+    )
+    target_lang = fields.Char(
+        "Target Language",
+        help="Shortened language code used as the target for the translation request.",
+        index=True,
+    )
```

Comment:
rather than 2 split index, it would be better with a common unique index on (message, target_lang), probably in that order